src/cache-provider.ts

```diff
@@ -26,7 +26,9 @@ export default async function createCacheProvider<Data = any, Error = any>({
   // Initialize database
   let db: IDBPDatabase
 
-  try {
```

Comment:
Please don't remove the `try...catch` block, it has been added here: #9 

Eventually it may be rewritten to this:
```ts
initDb().catch(error => 
  onError(error)
)
```

You should be able to reproduce exceptions by testing in Firefox Private mode, as described here: #5